### ISSUE TITLE
Add pixel data format to image scheama

### ISSUE THREAD

**Issue by rkm** [HicServices/DicomTypeTranslation#108]
We should consider adding tags which describe pixel data formats and compression to the image-level schema. This primarily refers to `TransferSyntaxUID` (and its human-readable description), which determines how the pixel data is encoded. Parsing this in fo-dicom can also be used to quickly determine if an image has been [lossy compressed](https://github.com/fo-dicom/fo-dicom/blob/0d8a1948371baee62856dec91328514152294f1b/DICOM/DicomTransferSyntax.cs#L203).

We already have `LossyImageCompression`, however this only takes boolean values of 0/1 instead of describing the compression format.

**Comment by jas88** [comment]
Good point, it would also be interesting to know just how the formats break down in the current dataset between each option. (Do we have lots of TIFFs for example?) We get MPEGs from the Cardiology PACS, it would be interesting to see if any of those have made it over to the National PACS yet...
